client/components/data/query-contact-details-cache/index.jsx

```diff
@@ -29,6 +33,9 @@ QueryContactDetailsCache.propTypes = {
 };
 
 export default connect(
-	( state ) => ( { isRequesting: isRequestingContactDetailsCache( state ) } ),
+	( state ) => ( {
+		contactDetailsCache: getContactDetailsCache( state ),
```

Comment:
If I'm reading correctly, the contents of the cache is never read in the component — we only care whether it's empty. In that case, I'd have:

```js
( state ) => ( {
  isContactDetailsCacheEmpty: isEmpty( … ),
  isRequesting: …,
} )
```